ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthorizationError.scala

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.auth
+
+import java.time.Instant
+
+sealed abstract class AuthorizationError {
```

Comment:
Instead of prepending each implementation with `AuthorizationError`, I would recommend putting them all in a companion object.